app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
Giải thích code `&.` cho chị nghe nào

Reply:
&. là 1 toán tử an toàn cho phép gọi một phương thức trên một đối tượng có thể là nil như try. (&.) sẽ đưa ra exception nếu 1 phương thức thực sự không tồn tại được gọi, trong khi try() luôn luôn trả về nil nên sử dụng (&.) sẽ hợp lý hơn.